roxctl/common/tls.go

```diff
@@ -56,6 +56,12 @@ func ConnectNames() (string, string, bool, error) {
 	if err != nil {
 		return "", "", false, errors.Wrap(err, "could not get endpoint")
 	}
+	if flags.UseKubeContext() {
```

Comment:
Shouldn't this at the top as this overrides endpoint provided in flag or maybe the two options should be mutually exclusive?

Reply:
It shouldn't. It is here to pick up the `usePlaintext` option.
[c.MarkFlagsMutuallyExclusive(useKubeContextFlagName, "endpoint")](https://github.com/stackrox/stackrox/blob/3e3d93c0af004a3977c01b01b098b28c6d702864/roxctl/common/flags/endpoint.go#L82)